server/db/db.js

```diff
@@ -694,6 +694,33 @@ async function getLocationsFromClientId(clientId, pgClient) {
   }
 }
 
+async function numberOfStillnessAlertsIn24Hours(locationid, pgClient) {
+  const endDate = new Date()
+  const startDate = new Date(endDate.getTime() - 24 * 60 * 60 * 1000)
+  try {
+    const results = await helpers.runQuery(
+      'numberOfStillnessAlertsIn24Hours',
+      `
+      SELECT COUNT (*)
+      FROM sessions
+      WHERE alert_type='SENSOR_STILLNESS'
```

Comment:
I think we have a constant somewhere that can be used instead of hardcoding the string `SENSOR_STILLNESS`. Generally speaking, it's better to use a constant than hardcoding values because it means that if something changes, then it only needs to change in one place.

Reply:
That's very true! Thank you for pointing this out, I'll look for the constant